citi-onboarding/acervusej-onepage
# Fix necessary stylings

**Things I've done:**

- Fix distance between web and mobile sessions
- Normalize web and mobile font-sizes
- Normalize and correct spacing
- Add switch sections through header menu
- Add dynamic styling of each menu item
- Putting slight transition in changing from missions to values
- Slowdown "Our services" carousel
- Add favicon
- Remove test images
- Remove overflow when mobile menu is open
- Add smooth scrolling

**How to test it:**

- First download the node modules on the client and the server. If you have already done that, then:
- Change the values on .env.
- Go to client on the terminal (cd client) and put npm run start.
- On the browser write http://localhost:3000/.
- See if the entire page works and looks fine on both mobile and web view.
- Look at the code to see if it has everything that it should have.
